exasol_sagemaker_extension/autopilot_handler.py

```diff
@@ -44,4 +48,23 @@ def check_training_status(job_name: str):
         describe_response["AutoMLJobSecondaryStatus"]
 
 
+def deploy_endpoint(
```

Comment:
Do we want to make more flexible and allow the user to specify the candidate. We don't need to forward this option to the UDF itself, but might b good idea for further extensions.

Reply:
in order to that we need to get user known about candidates. Because candidate names does not make sense (en example of a best candidate's name: end2endmodelname-0700GKOw6fqiQovy-001-b2dc0963)

We might return the all candidates to user for further extension so that they might perform  custom experiments on candidates and select whichever they want to deploy. Should I open a ticket for that?